### ISSUE TITLE
Installation process hang at "Setting up networking" step on Thinkpad T430 with HeadsOS

### ISSUE THREAD

**Issue by almazyan** [QubesOS/qubes-issues#8002]
![IMG_20230127_093738_487](https://user-images.githubusercontent.com/10381464/215064938-80e11031-760e-4155-ba80-60d30299954d.jpg)
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release
Qubes-R4.1.1-x86_64

### Brief summary
During installation process my pc hang at "Setting up networking" step (see image attached)

### Steps to reproduce
I followed installation instructions, and it freezed on the last step which is "[Dom0] Qubes OS Setup" - "Setting up networking"

### Expected behavior
It should have finished the installation process

### Actual behavior
The installation freezed on the last step which is "[Dom0] Qubes OS Setup" - "Setting up networking"

**Comment by andrewdavidwong** [comment]
Could you please provide details about your hardware?

**Comment by almazyan** [comment]
yes, it is Lenovo Thinkpad T430 with HeadsOS.
What else can I give you as a hardware detail?

I don't know if it is relevant but I obtained this bug when I chose sys-net and sys-usb in one configuration

**Comment by marmarek** [comment]
Is that from Nitrokey? You may need Heads update: https://docs.nitrokey.com/nitropad/qubes/firmware-update-1.4

**Comment by almazyan** [comment]
I don't know if it's from Nitrokey, I bought it from second hand.

Thanks for the link. It says "**If you try to update your Nitropad from a running system using a -maximized image, YOU WILL BRICK YOUR NITROPAD. The -maximized image must only be used with an external flasher device**". 
Since I don't have flasher device I won't take the risk to update it.

I've reinstalled Qubes without checking sys-net and sys-usb in one VM. There wasn't any problem.

**Comment by marmarek** [comment]
Yes, maximized firmware is a risky operation. But there is also non-maximized version. The differences are described on that page.

**Comment by almazyan** [comment]
> Yes, maximized firmware is a risky operation. But there is also non-maximized version. The differences are described on that page.

Thanks for the info.

In the system info window of my pc, it says "hotp-maximized":
![image](https://user-images.githubusercontent.com/10381464/215341543-02d4d0e0-d008-4d7d-b77c-60b8ba8507f1.png)

That's why, I recently downloaded a rom file "t430-hotp-maximized" from the site https://github.com/osresearch/heads and flashed into my BIOS via a usb key.

![image](https://user-images.githubusercontent.com/10381464/215340930-751cff20-1bd3-4f88-9039-bc9e068821ce.png)

What is the difference between the firmwares on the Nitrokey site you gave above and the ones from osresearch/heads on github?

**Comment by marmarek** [comment]
Ah, ok, so you have the maximized already, then keep using maximized :)
The version you took is a fresh build from most recent commit, while the Nitrokey 1.4 release seems to be based on [a version from about a year ago](https://github.com/Nitrokey/heads/commits/v1.4). Other differences are about Nitrokey branding, which shouldn't matter here.
In short: no, switching to Nitrokey version won't help, you have newer version already.

**Comment by almazyan** [comment]
I redownloaded Qubes and reinstalled with the configuration sys-usb and sys-net in one vm.
At the end of the installation I got this error message:
![image](https://user-images.githubusercontent.com/10381464/216188568-1e404ec9-5a5e-4ddb-9d5d-8a1887cd71e5.png)

What does it mean?

**Comment by marmarek** [comment]
sys-net failed to start. You can find more in the log mentioned in this message.

**Comment by almazyan** [comment]
I can't look inside the folder libvirt :/ "Permission denied".
FYI, I didn't configure root.
![image](https://user-images.githubusercontent.com/10381464/216364861-70804f18-703c-47c3-9fec-127aa2227d2e.png)

**Comment by almazyan** [comment]
Also it is strange that I don't have terminal in sys-net
![image](https://user-images.githubusercontent.com/10381464/216366806-32678867-007f-499b-aedf-cb9ef6090d24.png)

**Comment by andrewdavidwong** [comment]
> Also it is strange that I don't have terminal in sys-net ![image](https://user-images.githubusercontent.com/10381464/216366806-32678867-007f-499b-aedf-cb9ef6090d24.png)

You need to move one of the terminal entries (e.g., "Run Terminal") from the left-hand list to the right-hand list. The left-hand list shows everything installed (and detected) in the qube (or its template), whereas the right-hand list shows everything that has been selected to appear in the App Menu (which is the menu you have open on the far left of this image).

(@marmarta: Is it possible that the word "Available" makes some people think those are the things that are supposed to be *available* in the App Menu? Opened #8012 for your consideration.)

**Comment by SrEstegosaurio** [comment]
I've got the same error in a different machine _(ASUS ROG Strix G15 G513IM-HN161)_ and version 4.1.2-rc1 the installation process froze while setting up networking. It the whole device became unresponsive and needed to be rebooted. I tried several times with the same result, compared checksums and checked iso before installing.

I did a simple install, one of the few things that I iterated was making sys-net disposable. I'll try again but without that option enabled.
